### PR TITLE
Add card verification

### PR DESCRIPTION
**Pull request by OlegGS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +140,25 @@ public class Main {
...
+gw.process(verification);
+
+// send a payment with flag to accept only verified cards
+Command command = new Command().setCardVerification(CardVerificationMode.VERIFY);
+sms.setCustomer(customer).setCommand(command);
+```
+
 ### Requirements
 
 - This library works with Java 11 or above.
```
Comment (vglods) [inline README.md]: As I see, version was downgraded to Java 8, please fix doc

File: src/main/java/com/github/transactpro/gateway/model/request/data/CardVerificationModeSerializer.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.github.transactpro.gateway.model.request.data;
```
Comment (vglods) [inline src/main/java/com/github/transactpro/gateway/model/request/data/CardVerificationModeSerializer.java]: Definitely in wrong place

File: src/main/java/com/github/transactpro/gateway/model/request/data/CardVerificationMode.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.github.transactpro.gateway.model.request.data;
```
Comment (vglods) [inline src/main/java/com/github/transactpro/gateway/model/request/data/CardVerificationMode.java]: Should be under package "command"

File: src/main/java/com/github/transactpro/gateway/validation/VerifyCardGroup.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package com.github.transactpro.gateway.validation;
+
+import com.github.transactpro.gateway.validation.base.GwTransactionIdGroup;
+
+import javax.validation.GroupSequence;
+import javax.validation.groups.Default;
+
+@GroupSequence({Default.class, GwTransactionIdGroup.class})
+public interface VerifyCardGroup {
```
Comment (vglods) [inline src/main/java/com/github/transactpro/gateway/validation/VerifyCardGroup.java:9]: Could not find usage of this

**Review by vglods — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/transactpro/gateway/validation/VerifyCardGroup.java (hunk 4 above)
Comment (OlegGS) [reply src/main/java/com/github/transactpro/gateway/validation/VerifyCardGroup.java:9]: It is used in VerifyCard operation classin th same way EnrolledGroup is used.

File: README.md (hunk 1 above)
Comment (OlegGS) [reply README.md]: Changed

File: src/main/java/com/github/transactpro/gateway/model/request/data/CardVerificationModeSerializer.java (hunk 2 above)
Comment (OlegGS) [reply src/main/java/com/github/transactpro/gateway/model/request/data/CardVerificationModeSerializer.java]: Moved

File: src/main/java/com/github/transactpro/gateway/model/request/data/CardVerificationMode.java (hunk 3 above)
Comment (OlegGS) [reply src/main/java/com/github/transactpro/gateway/model/request/data/CardVerificationMode.java]: Moved

**Review by vglods — APPROVED** [review]

**Pull request merged**